package.json

```diff
@@ -50,15 +50,19 @@
     "react-helmet": "^3.1.0",
     "react-intl": "^2.1.5",
     "react-redux": "4.4.5",
-    "react-router": "^2.7.0",
-    "react-router-redux": "4.0.6",
     "redbox-react": "1.3.2",
     "redux": "^3.5.2",
+    "redux-little-router": "12.0.0",
     "redux-observable": "0.12.0",
     "request": "2.75.0",
     "rxjs": "^5.0.0-beta.11",
     "source-map-support": "0.4.3",
     "tough-cookie": "2.3.1",
     "url-search-params": "0.6.1"
+  },
+  "peerDependencies": {
```

Comment:
After reading more about how peerDependencies are supposed to be used, I think this is the correct way - basically, the platform is going to import some libs that it uses to set up the render, and the consumer is going to import the same libs for some of its rendering scripts, so the platform needs to make sure the same lib is being imported in both places by specifying them as `peerDependencies`.

Without this, the server-side `dom.jsx` might be rendered with React version 15 and the client will supply an `<App>` that renders with React version 20, which would be a problem.

An alternative that I've experimented with is for the platform to be the _only_ part of the system that imports React, and then it can wrap it for export so that consumers always use the same lib that the platform is using. That would mean using `import React from 'meetup-web-platform/lib/react';`, which is a little weird, but would enforce consistency.